app/assets/javascripts/discourse/app/templates/components/bookmark-list.hbs

```diff
@@ -0,0 +1,79 @@
+{{#conditional-loading-spinner condition=loading}}
+  {{#load-more selector=".bookmark-list .bookmark-list-item" action=(action "loadMoreBookmarks")}}
```

Comment:
can't you just do:

```
action=loadMore
```

?

Reply:
Ah...could have sworn I tried that. Thank you!